src/test/java/com/epam/prejap/teatrees/block/IBlockTest.java

```diff
@@ -0,0 +1,47 @@
+package com.epam.prejap.teatrees.block;
+
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+/**
+ * @author Marta Krawczykowska, Krzysztof Dolanski
+ */
+public class IBlockTest {
+
+
+    @Test
+    public void imageShouldBeTheSameAsGivenGeometricalFigure(){
+
+        // Given
+        Block iBlock = new IBlock();
+        // When
+        int i0j0 = 0;
```

Comment:
Couldn't this be written in a more compact way, say, with some loops?

Reply:
Done by usage of @DataProvider